math/mathcore/inc/Fit/FitExecutionPolicy.h

```diff
@@ -1,9 +0,0 @@
-#ifndef ROOT_Fit_FitExecutionPolicy
-#define ROOT_Fit_FitExecutionPolicy
-namespace ROOT{
-   namespace Fit{
-      enum class ExecutionPolicy { kSerial, kMultithread, kMultiprocess };
-    }
-}
```

Comment:
Hi Xavi, I am not sure what @Axel-Naumann thinks about this, but I would try to avoid replicating in ROOT (with an incompatible interface) things that are part or will be part of the standard¹.

This should be a much easier backport than the troublesome `std::string_view`, and using the standard version has the advantage that ROOT algorithms can then forward the execution policy to std algorithms when that makes sense.

1. http://en.cppreference.com/w/cpp/algorithm/execution_policy_tag

Reply:
At this point I need to differentiate between multiprocessed and multithreaded (unsequenced) executions, and more that will come. There's certainly some ways we can do this besides creating our own execution policies, but each one I can think of has its flaws.